### ISSUE TITLE
Not a chat

### ISSUE THREAD

**Issue by TobbyReign** [geekr-dev/geekchat#5]
A chat which need page to be refreshed to receive the next message are not a chat.

**Comment by geekr-dev** [comment]
> A chat which need page to be refreshed to receive the next message are not a chat.

No, it doesn't need to refresh the page to receive the reply messages. you can run `npm run dev` to build frontend assets
